src/index.ts

```diff
@@ -1,6 +1,6 @@
 import { CloudEvent, Version } from "./event/cloudevent";
 import { ValidationError } from "./event/validation";
-import { CloudEventV03, CloudEventV03Attributes, CloudEventV1, CloudEventV1Attributes } from "./event/interfaces";
+export * from "./event/interfaces";
```

Comment:
No star imports please.

Reply:
I think it makes more sense to export all interfaces directly, what is the reason behind no star in this case?

Reply:
Yeah, this looks fine actually!